### PR TITLE
Including the email documentation to the main doc link

### PR DESCRIPTION
**Pull request by chamindi-a**

### CODE REVIEW

**Review by nilminiwso2 — DISMISSED** [review]

**Review by nilminiwso2 — DISMISSED** [review]

File: en/docs/configurations/email-notification.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Configurations for Enabling Email Notifications
+
+This page documents the configurations that should be added in order to enable email notifications to the file deployment.toml under <code>&lt;OH_APIM_HOME&gt;/repository/conf</code>.
```
Comment (nilminiwso2) [inline en/docs/configurations/email-notification.md]:
```suggestion
This page documents the configurations that should be added in order to enable email notifications to the `deployment.toml` file under <code>&lt;OH_APIM_HOME&gt;/repository/conf</code>.
```

Comment (nilminiwso2) [reply en/docs/configurations/email-notification.md]: `<code>&lt;OH_APIM_HOME&gt;/repository/conf</code>` - why html code here?
we can use `<OH_APIM_HOME>/repository/conf` isnt't it

File: en/docs/get-started/open-healthcare.md [hunk 2]
```diff
@@ -1 +1,16 @@
-What is Open Healthcare
\ No newline at end of file
+
+
+The WSO2 Open Healthcare platform is a solution built on top of the best in its industry, that been an open-source integration platform. The platform allows you to connect to disparate source systems (e.g., EHRs/EMRs, Claim Management Systems, Databases etc.), quickly transforming your data to/from healthcare message formats (e.g., FHIR, HL7, X12, CCDA etc.) and expose them through secure APIs.<br> Our main intent is to accelerate the development process of healthcare IT through auto generative, easily extensible and well supported set of healthcare accelerators.
```
Comment (nilminiwso2) [inline en/docs/get-started/open-healthcare.md:3]: built on top of the best what?

File: en/docs/configurations/email-notification.md (hunk 1 above)
Comment (chamindi-a) [reply en/docs/configurations/email-notification.md]: when the normal <> tags are applied the tags and the content between is not seen
image (1)
 please see above image as an example

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**